lib/taskana-core/src/main/java/pro/taskana/configuration/TaskanaEngineConfiguration.java

```diff
@@ -417,12 +425,19 @@ public void setClassificationTypes(List<String> classificationTypes) {
         this.classificationTypes = classificationTypes;
     }
 
-    public List<String> getClassificationCategories() {
+    public List<String> getClassificationCategoriesByType() {
```

Comment:
This should rather be names getAllClassificationCategories, shouldn't it?

Reply:
yes, you are right. This method returns all the classification categories.
Thank you